angular-workspace/projects/ni/nimble-angular/src/directives/table/nimble-table.directive.ts

```diff
@@ -24,7 +24,9 @@ export class NimbleTableDirective<TData extends TableRecord = TableRecord> imple
         this.dataObservable = value;
```

Comment:
Can we enable selection in the angular and blazor example apps?

Reply:
I was planning on doing that in a follow up PR in an effort to keep this one from getting even larger than it already is. Are you fine with that plan, or do you think it would be beneficial to add support for selection now in order to validate some of the other changes I'm making?

Reply:
Follow-up works for me!